Compiler/Properties/launchSettings.json

```diff
@@ -11,8 +11,17 @@
     },
     "Multiple Errors": {
       "commandName": "Project",
-      "commandLineArgs": "--check ShouldFail/multiple_errors.bop",
+      "commandLineArgs": "--log-format JSON --check ShouldFail/multiple_errors.bop",
       "workingDirectory": "C:\\Users\\Frob\\Projects\\bebop\\Laboratory\\Schemas"
+    },
+    "Compile Files": {
+      "commandName": "Project",
+      "workingDirectory": "C:\\Users\\Frob\\Projects\\bebop\\Laboratory\\Schemas"
+    },
+    "Oops All Failures": {
+      "commandName": "Project",
+      "commandLineArgs": "--check",
+      "workingDirectory": "C:\\Users\\Frob\\Projects\\bebop\\Laboratory\\Schemas\\ShouldFail"
```

Comment:
These should be relative.

Reply:
Yeah I would have done that, but I'm not sure exactly what it's relative TO and I couldn't find any info on it. Can you enlighten me?